src/collective/volto/formsupport/restapi/services/submit_form/post.py

```diff
@@ -276,8 +282,25 @@ def send_data(self):
             self.send_mail(msg=msg, charset=charset)
 
     def prepare_message(self):
+        template_name = self.email_format_page_template_mapping[
+            self.block.get("email_format", "")
+        ]
+
+        if not template_name:
```

Comment:
This shouldn't be an option because we always have a default value

Reply:
I mean something like this:

```suggestion
 email_format_page_template_mapping = {
        "list": "send_mail_template",
        "table": "send_mail_template_table",
    }
 
 ...
 
 email_format = self.block.get("email_format", "")
 template_name = self.email_format_page_template_mapping.get(email_format, 'send_mail_template')
 
```

In this case you always have a default value (send_mail_template)